exercises/hello-world/hello_test.go

```diff
@@ -2,25 +2,17 @@ package greeting
 
 import "testing"
 
-// Define a function HelloWorld(string) string.
+// Define a function HelloWorld() string.
```

Comment:
Very nice point, I'll take it.

Another possibility - instead of speaking in terms of the signature, describe the signature in words? "Define a function named Hello World that takes no arguments and returns a string"? Better or worse?